ci/release/changelogs/next.md

```diff
@@ -1,5 +1,6 @@
 #### Features 🚀
 
+- `double-circle` keyword implemented. [#565](https://github.com/terrastruct/d2/pull/565)
```

Comment:
just realized this is `double-border` now

Reply:
oh right, damnit